test/Groups/libraries.jl

```diff
@@ -139,7 +139,8 @@ end
 
 @testset "Small groups" begin
    L = all_small_groups(8)
-   LG = [abelian_group(PcGroup,[2,4]), abelian_group(PcGroup,[2,2,2]), cyclic_group(8), quaternion_group(8), dihedral_group(8)]
+#  LG = [abelian_group(PcGroup,[2,4]), abelian_group(PcGroup,[2,2,2]), cyclic_group(8), quaternion_group(8), dihedral_group(8)]
```

Comment:
Is there a point in keep the "old" copies of the test around, here and below? Otherwise just delete them?

Reply:
I am still confident that eventually we can support `abelian_group(PcGroup, [2, 4])`.
I will add a comment.